contracts/KlerosPOC.sol

```diff
@@ -296,7 +311,7 @@ contract KlerosPOC is Arbitrator {
         uint amountShift=(alpha*minActivatedToken)/ALPHA_DIVISOR;
         for (uint i=0;i<=dispute.appeals;++i) {
             // If the result is not a tie, some parties are incoherent. Note that 0 (refuse to arbitrate) winning is not a tie.
-            if (winningChoice!=0 || (dispute.voteCounter[dispute.appeals].voteCount[0] == dispute.voteCounter[dispute.appeals].winningCount)) {
+            if (winningChoice!=0 || (dispute.voteCounter[dispute.appeals].voteCount[0] != dispute.voteCounter[dispute.appeals].winningCount)) {
```

Comment:
This was a bug in previous version of contract. Should be winning vote is not a tie OR a tie vote count is _not_ the winning count